apps/src/gamelab/GameLabP5.js

```diff
@@ -3,6 +3,7 @@ import {allAnimationsSingleFrameSelector} from './animationListModule';
 var gameLabSprite = require('./GameLabSprite');
 var gameLabGroup = require('./GameLabGroup');
 var Spritelab = require('./spritelab/Spritelab');
+import {backgrounds} from './spritelab/backgrounds.json';
```

Comment:
Cool, TIL you can import .json objects directly 👍

(Import backgrounds.json even for Game Lab levels, but luckily it's small and we'll fix when we separate out spritelab.js)